setup.cfg

```diff
@@ -43,6 +43,7 @@ install_requires =
     fasteners
     h5py ~= 2.9
     joblib
+    pycryptodomex
```

Comment:
why this one is needed? I do not see it (or `Crypto`) imported -- could comment be added right here?

Reply:
It's for the EncryptedKeyring backend.

Reply:
Comment added.